### PR TITLE
add libreoffice tc#1503906 script

### PR DESCRIPTION
**Pull request by dehai**
add  tc#1503906 :LibreOffice: Favorite Documents link in Computer menu. script  and modify main.pm to load test

### CODE REVIEW

File: tests/x11regressions/libreoffice/libreoffice_mainmenu_favorites.pm [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+use base "x11test";
+use strict;
+use testapi;
+
+sub run() {
+    my $self = shift;
+
+    # start destop application memu
+    wait_idle;
```
Comment (coolo) [inline tests/x11regressions/libreoffice/libreoffice_mainmenu_favorites.pm]: please avoid wait_idle - assert the screen you want to see. wait_idle sleeps a minute on real hardware, which is very expensive.

File: tests/x11regressions/libreoffice/libreoffice_mainmenu_favorites.pm [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2016 SUSE LLC
```
Comment (coolo) [inline tests/x11regressions/libreoffice/libreoffice_mainmenu_favorites.pm]: as you didn't work on this during 2012-2015, please limit the years of the copyright

File: tests/x11regressions/libreoffice/libreoffice_mainmenu_favorites.pm (hunk 1 above)
Comment (dehai) [inline tests/x11regressions/libreoffice/libreoffice_mainmenu_favorites.pm]: could I use the API of 'wait_still_ screen'?

File: tests/x11regressions/libreoffice/libreoffice_mainmenu_favorites.pm (hunk 2 above)
Comment (dehai) [inline tests/x11regressions/libreoffice/libreoffice_mainmenu_favorites.pm]: I will correct my year number , to only 2016.

File: tests/x11regressions/libreoffice/libreoffice_mainmenu_favorites.pm (hunk 1 above)
Comment (coolo) [inline tests/x11regressions/libreoffice/libreoffice_mainmenu_favorites.pm]: that is the preferred solution, yes

**Pull request merged**